maestro-cli/src/main/java/maestro/cli/command/PrintHierarchyCommand.kt

```diff
@@ -44,6 +48,14 @@ class PrintHierarchyCommand : Runnable {
 
     override fun run() {
         MaestroSessionManager.newSession(parent?.host, parent?.port, parent?.deviceId) { session ->
+            Insights.onInsightsUpdated {
+                val message = StringBuilder()
+                message.append("Warning!".yellow() + ": ")
+                it.first().message.chunkStringByWordCount(12).forEach { chunkedMessage ->
```

Comment:
This will only ever show the first insight even if there are multiple

Reply:
Done, fixed that by having single insight